.travis.yml.in

```diff
@@ -109,9 +116,11 @@ script:
           -DLEAN_EXTRA_MAKE_OPTS=$LEAN_EXTRA_MAKE_OPTS
           ../src || exit
   - make -j2 || exit
+  - export PATH="$TRAVIS_BUILD_DIR/bin:$PATH"
```

Comment:
@gebner `leanpkg` currently needs `lean` in its PATH, should we change this to use the one in its directory?